.github/workflows/kind-e2e.yaml

```diff
@@ -173,8 +172,15 @@ jobs:
       working-directory: ./src/knative.dev/serving
       if: matrix.kingress == 'istio'
       run: |
-        # TODO(https://github.com/knative-sandbox/net-istio/issues/285): Istio + NodePort
-        exit 1
+        set -o pipefail
```

Comment:
Do we need to set $NODE_PORT here ?

Reply:
Yes, good catch, also `local` is not allowed.